sdk/extendedlocation/Microsoft.Azure.Management.ExtendedLocation/src/Microsoft.Azure.Management.ExtendedLocation.csproj

```diff
@@ -6,11 +6,11 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.Management.ExtendedLocation</PackageId>
     <Description>Provides Microsoft Azure ExtendedLocation operations.</Description>
-    <Version>1.0.0-beta.2</Version>
+    <Version>1.0.0</Version>
```

Comment:
Confirming Version bump from 1.0.0-beta.2 to 1.0.0 as current is a GA API Version

Reply:
Confirmed?

Reply:
Is there a doc for Version number guidelines?

Reply:
Remove beta.x suffix once the lib is ready for GA

Reply:
Then this version number should be good. Thank you!